Userland/Games/Chess/PromotionDialog.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, the SerenityOS developers.
+ * Copyright (c) 2024, the SerenityOS developers.
```

Comment:
You are supposed to keep the old date and update it like this:
```
Copyright (c) 2020-2024, the SerenityOS developers.
```

Reply:
Fixed